frontend/src/components/onboarding-steps/InviteUsersStep.tsx

```diff
@@ -20,17 +31,19 @@ export default ({
         </h2>
 
         <div className="space-y-4">
-          <TextDivisor text="Emails*" />
+          <TextDivisor
+            text={`E-mails ${isEmailRequired || !isUserEmpty ? "*" : ""}`}
+          />
           <MultipleSelector
             removeSuggestions
             hidePlaceholderWhenSelected
             transformInputValueInLowercase
-            onChange={(x) =>
-              form.setValue(
+            onChange={(x) => {
+              return form.setValue(
                 "newUsersEmailsToInvite",
-                Array.from(new Set(x.map((x) => x.value)))
```

Comment:
Why did you remove the set of this line? This set was there to prevent repeated emails from being sent on the field.

Reply:
fixed @Gutts-n

Reply:
@Gutts-n fixed